docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+            "is_training" : false <7>
+          }
+----
+<1> The `dependent_variable` with the ground truth value. This is what we are 
+trying to predict with the {classanalysis}.
+<2> The top classes object contains the predicted classes with the highest 
+probability.
+<3> The probability that `FlightDelay` is false (the larger probability of the 
+two, hence this is the predicted class). This is a value between 0 and 1. The 
+higher the number, the higher the probability that the datapoint belongs to the 
+class.
+<4> Name of the class.
+<5> The probability that `FlightDelay` is true.
+<6> The prediction. The field name is suffixed with `_prediction` by default. 
+You can specify the field name by defining `prediction_field_name` via the API. 
+<7> Indicates that this document was not used in the training set.
+
+The example above shows that the analysis has predicted the probability of all 
+possible classes. In this case, there are two classes: `true` and `false`. The 
+class names along with the probability of the given classes are displayed in the 
+`top_classes` object. The most probable class is the prediction. In the example 
+above, `false` has a `class_probability` of 0.93 while `true` has only 0.06, so 
+the prediction will be `false` which coincides with the ground truth contained 
+by the `FlightDelay` field. The class probability values help you understand how 
+sure the model is about the prediction. The higher number means that the model 
+is more confident.
+
+If a document doesn't contain a prediction field, then it is excluded from the 
+analysis. In order to be analyzed, a document must contain at least one field 
+with a supported data type (`numeric`, `boolean`, `text`, `keyword` or `ip`) and 
+must not contain arrays with more than one item.
+--
+
+
+[[flightdata-classification-evaluate]]
+==== Evaluating results
+
+The results can be evaluated for documents which contain both the ground truth 
+field and the prediction. In the example below, `FlightDelay` contains the 
+ground truth and the prediction is stored as `FlightDelay_prediction`.
+
+. Use the {dfanalytics} evaluate API to evaluate the results.
++
+--
+First, we want to know the training error that represents how well the model 
+performed on the training dataset:
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{
+ "index": "df-flight-delayed",  <1>
+   "query": {
+    "term": {
+      "ml.is_training": {
+        "value": true  <2>
+      }
+    }
+  },
+ "evaluation": {
+   "classification": {
+     "actual_field": "FlightDelay",  <3>
+     "predicted_field": "ml.FlightDelay_prediction",  <4>
+     "metrics": {  
+       "multiclass_confusion_matrix" : {}
+     }
+   }
+ }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> The destination index which is the output of the analysis job.
+<2> We calculate the training error by only evaluating the training data.
```

Comment:
> We calculate the training error by only evaluating the training data...

I think this is important enough to be moved out of the footnote.  For example, noting that in previous steps we saw that the new index contained a field that indicated which documents were used as training data, which we can now use to calculate the training error.